libosinfo.rb

```diff
@@ -0,0 +1,91 @@
+class Libosinfo < Formula
+  desc "The Operating System information database"
+  homepage "https://libosinfo.org/"
+  url "https://fedorahosted.org/releases/l/i/libosinfo/libosinfo-0.3.0.tar.gz"
+  sha256 "538a3468792e919edf5364fe102d751353ae600a92ad3a24f024424a182cefbc"
+
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "wget" => :build
+
+  depends_on "check"
+  depends_on "glib"
+  depends_on "libsoup"
+  depends_on "libxml2"
+  depends_on "libxslt"
+
+  depends_on "gobject-introspection" => :optional
+  depends_on "vala" => :optional
+
+  # work around "ld: unknown option: --no-undefined"
+  # upstream bug: https://bugzilla.redhat.com/show_bug.cgi?id=1305016
+  patch :DATA
+
+  def install
+    args = ["--prefix=#{prefix}",
```

Comment:
Another style nit: Prefer `args = %W[]` style arrays. Feel free to ignore the two style nits if there's nothing else to fix though; I can grab them after we merge if necessary.